### PR TITLE
Update to Android SDK 12.0.1

### PR DESCRIPTION
**Pull request by samtstern**

### CODE REVIEW

File: common/pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>com.google.firebase</groupId>
       <artifactId>firebase-admin</artifactId>
-      <version>[5.3.0,)</version>
+      <version>[5.9.0,)</version>
```
Comment (vanniktech) [inline common/pom.xml]: I mean here.

Comment (samtstern) [reply common/pom.xml]: Oh yeah, you're right.  I was so used to Android/Gradle land where we constantly bump our dependencies that I just did this instinctively.  I'll revert it.

**Comment by samtstern** [comment]
Ok I found the issue.  The Android support library changed how they make the `pom.xml` files. 

Here are the dependencies for `support-v4` version `25.2.0`:

```xml
  <dependencies>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-compat</artifactId>
      <version>25.2.0</version>
      <type>aar</type>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-media-compat</artifactId>
      <version>25.2.0</version>
      <type>aar</type>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-core-utils</artifactId>
      <version>25.2.0</version>
      <type>aar</type>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-core-ui</artifactId>
      <version>25.2.0</version>
      <type>aar</type>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-fragment</artifactId>
      <version>25.2.0</version>
      <type>aar</type>
      <scope>compile</scope>
    </dependency>
  </dependencies>
```

And here are the dependencies for `support-v4` version `26.1.0`:

```xml
  <dependencies>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-compat</artifactId>
      <version>26.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-media-compat</artifactId>
      <version>26.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-core-utils</artifactId>
      <version>26.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-core-ui</artifactId>
      <version>26.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>support-fragment</artifactId>
      <version>26.1.0</version>
      <scope>compile</scope>
    </dependency>
  </dependencies>
```

You can see they're no longer specifying `<type>aar</type>`.  I will file a bug with the support lib people.

**Comment by samtstern** [comment]
Looks like the issue was fixed after 27.0.0:
https://b.corp.google.com/issues/67576406

**Comment by samtstern** [comment]
@vanniktech thanks!  Replace with public link.  Our internal proxy always redirects to the corporate version, leading to a lot of copy-paste blunders.

**Comment by vanniktech** [comment]
That works now, thanks.

**Pull request merged**